tests/cupyx_tests/fallback_mode_tests/test_fallback.py

```diff
@@ -1,8 +1,11 @@
+import os
 import sys
 import pytest
 import unittest
 import functools
 
+os.environ['CUPY_EXPERIMENTAL_SLICE_COPY'] = str(1)
```

Comment:
Do not write it here because slice copy is enabled also in unrelated tests.

Reply:
Okay, I'll move related tests to a new test file.

Reply:
You don't need to separate the test file. Could you add `setUp()` and `tearDown()` methods?

Reply:
Implemented a decorator to combine `setUp()` and `tearDown()`.